meta-genivi-dev/recipes-web/zebkit/zebkit_1.0.0+20170516.bb

```diff
@@ -0,0 +1,293 @@
+# Recipe created by recipetool
+# This is the basis of a recipe and may need further editing in order to be fully functional.
+# (Feel free to remove these comments when editing.)
```

Comment:
I agree with @chbae - you need to squash the commits together to one before we can review fully.  (There is a commit adding a long file, then one that removes it again)  

Just adding a few early review comments for you to look at also:

See on this line there is an instruction to remove the generated comments.